filesize.d.ts

```diff
@@ -70,10 +70,22 @@ declare namespace Filesize {
          * Output of function (array, exponent, object, or string), default is string
          */
         output?: "array" | "exponent" | "object" | "string";
+        /**
+         * Decimal place end padding, default is false
+         */
+        pad?: boolean;
+        /**
+         * Sets precision of numerical output, default is 0
+         */
+        precision?: number;
         /**
          * Decimal place, default is 2
          */
         round?: number;
+        /**
+         * 
+         */
+        roundingMethod?: "round" | "floor" | "ceil";
```

Comment:
This is a dupe of line 110; just going to move the description here and delete the other instance.